stories/molecules/SearchMultiSelectExtended.mdx

```diff
@@ -0,0 +1,121 @@
+
+As the name suggests, the `<SearchMultiSelectExtended>` comes with a case insensitive term matching, and a small tag indicator to show you how many options were selected, as well a small balloon where you can quickly see what options you have selected so far.
+
+Search-Multi-select also supports nested options [See Nested Options](#nested-options). When a parent option is selected, all children will be selected. If all children are selected, the parent will be selected too.
+
+## Options
+
+Options determine the construction of the multi-select menu. It takes an `Array` of `Objects` with the following shape:
+
+```js
+{
+    name: 'John', // label displayed next to the checkbox
+    value: 'john', // value associated with the option
+    isSelected: true // current selected state
+}
+```
+
+Here's an example of a possible set of options:
+
+```js
+const people = [
+    {
+        name: 'John',
+        value: 'john',
+        isSelected: true
+    },
+    {
+        name: 'Sam',
+        value: 'sam',
+        isSelected: false
+    },
+    {
+        name: 'Jane',
+        value: 'jane',
+        isSelected: true
+    },
+    {
+        name: 'Sara',
+        value: 'sara',
+        isSelected: false
+    }
+];
+```
+
+### Nested Options
+
+You may also have nested options. Here's an example:
+
+```js
+const pets = [
+    {
+        name: 'All Pets',
+        value: 0,
+        isSelected: false
+    },
+    {
+        name: 'Dogs',
+        value: 1,
+        isSelected: false,
+        options: [
+            {
+                name: 'Lassie',
+                value: 1,
+                isSelected: true
+            },
+            {
+                name: 'Snoopy',
+                value: 2,
+                isSelected: true
+            }
+        ]
+    },
+    {
+        name: 'Cats',
+        value: 'cats',
+        isSelected: false,
+        options: [
+            {
+                name: 'Grumpy Cat',
+                value: 1,
+                isSelected: true
+            },
+            {
+                name: 'Lil Bub',
+                value: 2,
+                isSelected: true
+            }
+        ]
+    }
+];
+```
+
+# Full Example
+```jsx
+
+const [pets, setPets] = useState(initialPetsObject);
+return <SearchMultiSelectExtended onChange={newPets => setPets(newPets)} options={pets} size="md" />;
+```
+
+
+## Keybindings
+| key   | description                              |
+| ----- | ---------------------------------------- |
+| esc   | closes the dropdown                      |
+## Component Example
+
+<Canvas of={SearchMultiSelectExtendedStories.Default} />
+
+## Component Example Source
+
+<Source of={SearchMultiSelectExtendedStories.Default} />
+
+## SearchMultiSelectExtended Props
+
+<ArgsTable of={SearchMultiSelectExtended} />
```

Comment:
it says ArgsTable is not found.. need to import it as well as Canvas and Source (see others for example)

Reply:
@belinda82 it's giving an error that SearchMultiSelectExtendedStories is not found (https://deploy-preview-117--gumdrops.netlify.app/?path=/docs/molecules-searchmultiselectextended--docs)

Reply:
This component can display hierarchical data or nested options.